File: primatis-api/src/main/java/be/primatis/article/dto/ArticleMediaUploadResponse.java
package be.primatis.article.dto;

/**
 * Réponse de l'upload d'une image destinée à {@code Article.content}
 * (DEV-ARTICLES-MEDIA). {@code url} est une URL publique relative
 * ({@code /media/articles/<uuid>.<ext>}) — jamais un chemin filesystem
 * (mission §9 : ne jamais exposer {@code /home/...}/{@code C:\...}).
 */
public record ArticleMediaUploadResponse(String url) {
}

File: primatis-web/src/app/articles/models/article-media-upload-response.ts
/**
 * Voir `be.primatis.article.dto.ArticleMediaUploadResponse` côté backend
 * (DEV-ARTICLES-MEDIA). `url` est une URL publique relative
 * (`/media/articles/<uuid>.<ext>`), jamais un chemin filesystem.
 */
export interface ArticleMediaUploadResponse {
  url: string;
}
